op-node/service.go

```diff
@@ -289,9 +289,19 @@ func NewSyncConfig(ctx *cli.Context, log log.Logger) (*sync.Config, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	//fastnode config
+	elTriggerGap := ctx.Int(flags.ELTriggerGap.Name)
+	if ctx.Bool(flags.FastnodeMode.Name) {
+		mode = sync.ELSync
+		// fastnode needs a smaller gap
+		elTriggerGap = 120
```

Comment:
Fastnode must rely on el sync whether this gap can be set to 0. Otherwise, can alt sync solve the synchronization problem within 120 gap?

Reply:
Setting it to 0 comes back to the state where engine-sync is fully enabled. When gossip broadcasting is unstable, engine-sync will be triggered but will only sync a small range, which is not what we want to see. Within a 120-gap, if alt-sync cannot complete, the gap will continue to increase. When it reaches 120, engine-sync will still be triggered, so this is not a concern.